cl/opinion_page/templates/view_opinion.html

```diff
@@ -83,6 +83,28 @@ <h3 class="inline"><span>Your Notes</span></h3>
             </div>
         </div>
 
+        {% if summaries_count > 0  %}
+        <div id="summaries" class="sidebar-section">
+          <h3><span>Summaries ({{ summaries_count|intcomma }})</span></h3>
+          <p class="bottom">Judge-written summaries of this case:</p>
+          <ul>
+            {% for summary in top_summaries %}
+              <li>
+                {{ summary.text }}
```

Comment:
Since this is unbound server-side, we should at least put a cap here client-side. There are template filters for this... `numwords` maybe? Something like that.

Reply:
would ~50 words be appropriate for a cap here?